app/models/content_page.rb

```diff
@@ -12,12 +12,24 @@ class ContentPage
 
   # @return [String]
   def heading
-    translate(:heading)
+    if module_item.confidence_intro? || module_item.assessment_intro? || module_item.ending_intro?
```

Comment:
`module_item.confidence_intro? || module_item.assessment_intro? || module_item.ending_intro?`

can move to a private method (these pages have static content)